service/frontend/service.go

```diff
@@ -188,6 +189,7 @@ func NewConfig(
 		PersistenceNamespaceMaxQPS:            dc.GetIntPropertyFilteredByNamespace(dynamicconfig.FrontendPersistenceNamespaceMaxQPS, 0),
 		PersistencePerShardNamespaceMaxQPS:    dynamicconfig.DefaultPerShardNamespaceRPSMax,
 		EnablePersistencePriorityRateLimiting: dc.GetBoolProperty(dynamicconfig.FrontendEnablePersistencePriorityRateLimiting, true),
+		PersistenceDynamicRateLimitingParams:  dynamicconfig.DisabledDynamicRateLimiting,
```

Comment:
frontend, matching, and worker should also be able to enable dynamic rate limiter?

Reply:
I thought for this iteration, we only wanted to do history, but I can add logic for the other three if we want to include those as well.

Reply:
yeah let's include them, they will be disabled by default anyway.